documentation/user_docs/docs/manual/Advanced_Multifit.rst

```diff
@@ -2,33 +2,73 @@
+- For sqw objects, the splitting occurs across pixels. (``N/P`` pixels per process)
+- For dnd objects, the splitting occurs across bins. (``B/P`` bins per process)
+- For IX_dataset objects, the splitting occurs across points. (``N/P`` points per process)
+
+If the fitting is run as ``-ave``, then the decomposition will not split bins, but will distribute whole bins only.
 
 Multifit methods requirements
 =============================
+This section describes how to set up an object to work with Multifit specifications
 
-In many cases, the most convenient thing to do is extract the x,y,e arrays from an object and pass those to multifit. This can be done by using a method defined on the object to extract an x,y,e triple:
+In many cases, the most convenient thing to do is extract the x,y,e arrays from an object and pass those to
+multifit. This can be done by using a method defined on the object to extract an x,y,e triple:
 
 ::
 
-        >> [wout, fitdata] = multifit (xye(w), func, pin,...)
+   >> [wout, fitdata] = multifit (xye(w), func, pin,...)
 
-where the method xye must return a structure of the form required by multifit, namely a structure with fields ‘x’,’y’ and ‘e’ ,where ‘x’ is a cell array {x1,x2,…} containing the coordinates of the points along the first, second… axes, and ‘e’ contains the standard deviations. A convenient way to do this is to use the methods sigvar_get and sigvar_getx if they have been written to allow the object itself to be passed to multifit (see below).
+where the method xye must return a structure of the form required by multifit, namely a structure with fields ``x``,
+``y`` and ``e`` ,where ``x`` is a cell array ``{x1,x2,...}`` containing the coordinates of the points along the first,
+second, ...  axes, and ``e`` contains the standard deviations. A convenient way to do this is to use the methods
+sigvar_get and sigvar_getx if they have been written to allow the object itself to be passed to multifit (see below).
 
-If multifit is being used to fit functions to objects rather than x-y-e triples, then there are some methods that need to be defined. You might want to fit the objects if their internal structure is more complex, for example if the fitting function depends on fields other than just the x values and parameters being passed to the fit function. Another case is when the masking of points from fitting requires manipulation of fields other than simply removing x-y-e values. [An example is the case of the sqw objects used in Horace. Here the calculation of the intensity at a data point depends on the information of the individual pixels that contribute to that data point. Masking requires that the pixel information of masked bins is removed from the sqw object.]
+If multifit is being used to fit functions to objects rather than x-y-e triples, then there are some methods that need
+to be defined. You might want to fit the objects if their internal structure is more complex, for example if the fitting
```

Comment:
"Need to be defined", presumably by the user, so I think we need to say so. As you follow this with sentences starting "you", maybe this one as well "you need to define"